src/igvfd/audit/file_set.py

```diff
@@ -123,7 +155,7 @@ def audit_inconsistent_seqspec(value, system):
     '''
     [
         {
-            "audit_description": "Sequence files in a file set from the same sequencing run, flowcell_id, lane, and index are expected to link to the same seqspec file, which should be unique to that set of sequence files.",
+            "audit_description": "Sequence files in a file set from the same sequencing set, are expected to link to the same seqspec file, which should be unique to that set sequencing set. A sequencing set is defined by the combination of flowcell ID, lane, and index for non-single cell data, or flowcell ID, lane, index, and sequencing run for single cell data.",
```

Comment:
Could we delete that first comma after "same sequencing set"? I think I typoed in my original comment, sorry! And I think I left in an extra "sequencing set" in "that set sequencing set".